src/reader-activation/index.js

```diff
@@ -174,14 +172,7 @@ export function openAuthModal( config = {} ) {
 		}
 		return;
 	}
-	if ( readerActivation._openAuthModal ) {
-		readerActivation._openAuthModal( config );
-	} else {
-		console.warn( 'Authentication modal not available' ); // eslint-disable-line no-console
-		if ( config.onError && typeof config.onError === 'function' ) {
-			config.onError();
-		}
-	}
+	_openAuthModal( config );
```

Comment:
I'm importing the openAuthModal function directly so it's always available on RAS initialization. This is required to trigger the auth modal on RAS init without a domReady race condition.